ices/85350.rs

```diff
@@ -0,0 +1,4 @@
+impl FnMut(&Context) for 'tcx {
+    fn print () -> Self :: Output{ }
+    }
+    
```

Comment:
```suggestion
}

```

The indent looks weird.